core/src/main/java/hudson/model/View.java

```diff
@@ -921,7 +921,7 @@ public AsynchPeople(View parent) {
                 JSONObject entry = new JSONObject().
                         accumulate("id", u.getId()).
                         accumulate("fullName", u.getFullName()).
-                        accumulate("url", u.getUrl()).
+                        accumulate("url", u.getUrl() + "/").
```

Comment:
```suggestion
                        accumulate("url", StringUtils.appendIfMissing(u.getUrl(), "/")).
```

Fortunately the class is already imported in View 🎉 
BUUUUUT, we are including **commons-lang:2.6** ❌ 

IOW, not a good reason to potentially break other stuff just for this quick change.